supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -11,37 +11,37 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
-==== image:images/yes.png[yes] IBM eServer System p (noun)
-*Description*: Use "IBM eServer System p" for the first reference, and "IBM System p" or "System p" for subsequent references.
+[[ibm-power]]
+==== image:images/yes.png[yes] IBM Power® (noun)
+*Description*: _IBM Power_ represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products. On first use, mark IBM Power with a (®).
```

Comment:
```suggestion
*Description*: _IBM Power_ represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products. On first use, mark "IBM Power" with a trademark icon (®).
```

Reply:
@ndeevy yes it should, good call.

Reply:
Ok, thanks Miriam!
 FYI @SNiemann15 that means there's a second one that needs `registered` added to Miriam's suggested `trademark icon` on line 38. Thanks :-]